### ISSUE TITLE
Incompatible Kim-Api and Kimpy

### ISSUE THREAD

**Issue by Rana-Phy** [openkim/kliff#56]
Dear Developers,

Really disappointed in myself. 
Would you kindly let me know which version of kliff will be compatible with which version of Kim-Api and Kimpy?

Best regards,
Rana

**Comment by Rana-Phy** [comment]
Did someone find this package really working?
I found all the examples giving errors!

**Comment by mjwen** [comment]
Hi @Rana-Phy, 

Thanks for raising the compatible issue. We will add a compatibility check to kliff like [this](https://github.com/openkim/kimpy/blob/master/api_compatibility.py) shortly to ensure correct dependance. But if you follow the [documentation](https://kliff.readthedocs.io/en/latest/), everything should work. 

I do want to say that we make the package open source and hope it may help others for their reseach. If you find bugs, please do raise an issue or make a PR, instead of complaining about it (this really hurts). If you follow the tutorials in the doc, I am sure they should work. Again, if not, please let us know.

**Comment by Rana-Phy** [comment]
Hi @mjwen 

Sorry. I really did not mean any complain. I just wanted to know if someone runing this package properly now then which version of kliff, api, kimpy they are using.
I really appreciate your work and can feel how hard task it is.

Best wishes, 
Rana

**Comment by mjwen** [comment]
Hi @Rana-Phy, 

No worries, I understand the feeling when you expect a package to work but it does not work as expected.

I've just followed the instructions in the [documentation](https://kliff.readthedocs.io/en/latest/) to install it and run examples, basically what I did was (I just created a conda environment `kliff_env` to start from scratch there):

```
conda create -n kliff_env    
conda activate kliff_env 
conda install -c conda-forge python=3.9 kliff pytorch
```

Basically, this will install `kliff-0.4.0`, `kimpy-2.0.0`, `kim-api-2.2.1`, and other necessary packages. 

After these, the [tutorial example](https://kliff.readthedocs.io/en/latest/auto_examples/example_nn_Si.html#sphx-glr-download-auto-examples-example-nn-si-py) can run without problem. 

If you encounter problems, you are welcome to provide more info, e.g. your operating system, python version... Then we can see what's going on there. 

Hope it helps.
